### ISSUE TITLE
Proposed Initial Working Groups and Outreach Groups

### ISSUE THREAD

**Issue by jpitts** [gitcoinco/ERC-1789#42]
In order to avoid the pitfalls of Dunbar's number, I propose the formation of small working groups (WGs), with significant overlap. 

Also, I propose the creation of advocacy / learning / outreach groups (OGs), perhaps regional, and these would have overlap with certain WGs as well. 

This proposal generally follows the idea of a "swarm organization". https://falkvinge.net/2013/02/14/swarmwise-the-tactical-manual-to-changing-the-world-chapter-one/

## Group Size and Self-Organizing

From the definition Dunbar's limit:

> Dunbar's number is a suggested cognitive limit to the number of people with whom one can maintain stable social relationships—relationships in which an individual knows who each person is and how each person relates to every other person.

From the Swarmwise tactical manual:

> You will probably have a couple of formal groups that are about thirty people in size, like the assembled group of all officers and leaders for a certain function or geography, but in general, **you should strive for the seven-person group**. 

Source: https://falkvinge.net/2013/04/01/swarmwise-the-tactical-manual-to-changing-the-world-chapter-three/

There is a limit to how big groups can be in order to make decisions. Centralization of authority is one solution, but we are generally anti-authoritarian. This means we need highly independent teams, with a strong vision/mission consensus and social overlap in order to maintain cohesion.

## Proposed Working Groups - WGs

Working groups should have around 7 core members making decisions and doing the work.

**Initial WGs:**

- Vision and Mission / principles
- Marketing / messaging
- Political outreach / diplomacy
- Operations
- Block reward mechanism
- Governance of allocation / signaling
- Allocation mechanism / DAOs
- Accountability framework / measurement
- Coordination

These groups should be essentially autonomous and self-organizing, but working hard to follow the principle of coordination in good faith. 

_Question: should reps from each WG form a larger group called Coordination? Or should Coordination be a very small team?_

**The current work:**

- "Vision" would create the manifesto. 
- "Political" would be forming the EIPs, as these are to go to the core devs. 
- Most of the other WGs would be doing the R&D to help "Political" form that EIP. They depend heavily on each other at this point.
- "Marketing" would find a number of different ways to pitch the results, coordinating with "Political" to make it happen. 

Then the Outreach Groups would get into the Meetups and elsewhere, help make this happen on all levels.

## Proposed Outreach Groups - OG

The outreach groups will take in and foster leadership for most people getting involved from this point forward. They will be regional, let those in the Telegram channel decide which ones they want to go in on, but do not let them get too large. Once that happens, if that happens, then they can split.

**Initial OGs:**

- Central/West US
- East/South US
- Central Europe
- East Europe - needs more participants
- East Asia - needs more participants
- Southeast Asia - needs more participants

_Question: might this be better organized around "stakeholder groups" instead?_

**Overlap:**

The following WGs would have significant overlap w/ the OGs:

- Marketing / messaging
- Political outreach / diplomacy

---

**Record of UPDATES**

4/3/2019 - Added "Coordination". Inspired by the ElectronJS governance page and its notion of an "Administrative WG" (SEE: https://electronjs.org/governance).

4/3/2019 - Improved the naming of the WGs to better reflect the current proposal and diagram.

**Comment by pet3r-pan** [comment]
Might be able to help with Southeast Asia, have contacts to HK & Shanghai communities. Also know someone in the core community that resides in Tokyo. Korea, have a link there.

Also for the working groups, maybe that could be run similar to a Holocracy, similar to how Giveth distributes their roles & responsibilities (except made super simple in the beginning for now).

**Comment by glauseWilde** [comment]
Given that there are multiple implementation proposals in play do you envision each having its' own WG? Or would the Vision WG be representatives from each participant? @jpitts #36 <- discussed multi-proposal approach here.

Also, I really like looking at this from a Dunbar limit perspective. I am going to add some of that to my proposal.

**Comment by pet3r-pan** [comment]
We would need to implement this gradually and as needed to ensure we don't increase pointless surface area of attention.

**Comment by jpitts** [comment]
Activity should over time collect around these proposed WGs, even if there are different proposals in play. It is OK for that to happen, R&D needs to explore, but at some point choices will have to me made (disagree and commit).

Definitely agree about the gradualism mentioned by @pet3r-pan . 

The Vision WG in particular needs to have a simple, clear vision definition and mission statement. The manifesto takes these and supports them with evidence and a sense of urgency. 

I expect Vision in particular to adjust early-on, as the R&D happening in Rewards and in Distribution finds their optimal paths.

**Comment by owocki** [comment]
i propose that we find a way to self-organize into these working groups in the next week or two.  maybe we recruit "working group captains" that are all responsible for coordinateing / recruiting for their working groups.

**Comment by jpitts** [comment]
Changing some of the terminology to better match what I am seeing in the proposal / diagram.

**Comment by jpitts** [comment]
Here is a link to the initial signups for the WGs: 
https://docs.google.com/document/d/1yXs59MoRJ24prRocwbS3CQIcO1JdyDGP8sv5Bk7Fg3M/edit#bookmark=id.iru3jk7f7l0f

It was mentioned in the call that these should have at least one person taking responsibility for the WG's progress.
